menu.php

```diff
@@ -10,16 +10,25 @@
 <body>
     <div class="header">
         <div class="logo">
-            <a href="home.php">
-                <img src="images/starbullslogo.png" alt="logo" /></a>
+        <a href="home.php" class="logo">
+				<img src="images/starbullslogo.png" alt="logo" />
+			</a>
         </div>
+
         <h1>STARBULLS
             <div class="topnav">
                 <a href="menu.php">Menu</a>
                 <a href="specialMenu.php">Specials Of The Week</a>
                 <a href="about.php">About Us</a>
                 <a href="apply.php">Apply Now</a>
             </div>
+            <nav>
+            <a href="#breakfastSingles">Breakfast Singles</a>
+            <a href="#breakfastCombo">Breakfast Combos</a>
+            <a href="#tradWings">Our Wings</a>
+            <a href="#dinner">Dinner</a>
+            <a href="#sides">Sides</a>
+            </nav>
```

Comment:
this isn't the main nav and should not be wrapped in a `<nav>` tag... that should be used for main navigation